cmd/uplink/cmd/setup_test.go

```diff
@@ -115,3 +121,58 @@ func TestDefaultPortAppliedToSatelliteAddrWithPortColonButNoPort(t *testing.T) {
 	assert.Equal(t, "ahost:7777", setupCmd.Flags().Lookup("satellite-addr").Value.String(),
 		"satellite-addr should contain default port when no port specified")
 }
+
+func TestSaveEncriptionKey(t *testing.T) {
+	expectedKey := make([]byte, rand.Intn(20)+1)
+	_, err := rand.Read(expectedKey)
+	require.NoError(t, err)
+
+	t.Run("ok", func(t *testing.T) {
+		ctx := testcontext.New(t)
+		defer ctx.Cleanup()
+
+		fname := ctx.File("storj-test-cmd-uplink", ".enc.key")
+		err = cmd.SaveEncryptionKey(expectedKey, fname)
+		require.NoError(t, err)
+
+		// Read the key from the file to compare that it matches with the saved one.
+		f, err := os.Open(fname)
```

Comment:
ioutil.ReadFile

Reply:
The file is needed for the following check.